applications/GeoMechanicsApplication/custom_utilities/stress_strain_utilities.h

```diff
@@ -39,9 +42,25 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) StressStrainUtilities
                                                 bool                       UseHenckyStrain,
                                                 std::size_t                VoigtSize);
 
+    static constexpr std::size_t GetVoigtSize(std::size_t Dimension)
+    {
+        return Dimension == N_DIM_3D ? VOIGT_SIZE_3D : VOIGT_SIZE_2D_PLANE_STRAIN;
+    }
+
+    static constexpr std::size_t GetStressTensorSize(std::size_t Dimension)
+    {
+        return Dimension == N_DIM_3D ? STRESS_TENSOR_SIZE_3D : STRESS_TENSOR_SIZE_2D;
+    }
+
+    static const Vector& GetVoigtVector(std::size_t Dimension);
+
+    static const Vector VoigtVector2D;
+    static const Vector VoigtVector3D;
```

Comment:
I think it's better if these are private, such that we only reach them through the `GetVoigtVector` function